### ISSUE TITLE
Border doesn't update when resizing windows using keyboard

### ISSUE THREAD

**Issue by freerig** [hyprwm/Hyprland#7746]
### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Regression?

Yes

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit 9a09eac79b85c846e3a865a9078a3f8ff65a9259  (props: bump version to 0.42.0).
Date: Wed Aug 7 19:17:10 2024
Tag: v0.42.0, commits: 5069

flags: (if any)


System Information:
System name: Linux
Node name: **Hmmm private bro**
Release: 6.10.8-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Wed, 04 Sep 2024 15:16:37 +0000


GPU information: 
01:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] [1002:67df] (rev e7) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:
  hyprexpo by Vaxry ver 1.0
  hyprwinwrap by Vaxry ver 1.0

======Config-Start======


Config File: /home/quelquun/.config/hypr/hyprland.conf: Read Succeeded

# #######################################################################################
# AUTOGENERATED HYPR CONFIG.
# PLEASE USE THE CONFIG PROVIDED IN THE GIT REPO /examples/hypr.conf AND EDIT IT,
# OR EDIT THIS ONE ACCORDING TO THE WIKI INSTRUCTIONS.
# #######################################################################################

#
# Please note not all available settings / options are set here.
# For a full list, see the wiki
#

autogenerated = 0 # remove this line to remove the warning

# See https://wiki.hyprland.org/Configuring/Monitors/
monitor=DVI-D-1,preferred,auto,auto,transform,0
monitor=HEADLESS-2,2048x1536,auto,auto


# See https://wiki.hyprland.org/Configuring/Keywords/ for more

# Execute your favorite apps at launch
# exec-once = waybar & hyprpaper & firefox

# Source a file (multi-file configs)
# source = ~/.config/hypr/myColors.conf

# Set programs that you use
$terminal = kitty --single-instance
$browser = chromium # bye... thorium-browser
$fileManager = kitty --single-instance lf
$menu = $HOME/.config/scripts/app_launcher.py
$mainChat = signal-desktop

exec-once = xdg-open https://youtu.be/l0qvxPPISuY?t=10


# Some default env vars.
env = XCURSOR_SIZE,24
env = WLR_NO_HARDWARE_CURSORS,1
env = QT_QPA_PLATFORMTHEME,qt5ct # change to qt6ct if you have that
# Dark mode
env = GTK_THEME,Adwaita:dark
env = GTK2_RC_FILES,/usr/share/themes/Adwaita-dark/gtk-2.0/gtkrc
env = QT_STYLE_OVERRIDE,Adwaita-Dark
env = XKB_DEFAULT_LAYOUT,fr

# For all categories, see https://wiki.hyprland.org/Configuring/Variables/
input {
    kb_layout = fr, us
    kb_variant =
    kb_model =
    kb_options =
    kb_rules =

	numlock_by_default = true

    follow_mouse = 1

    sensitivity = 0.1 # -1.0 to 1.0, 0 means no modification.
}

general {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more

    gaps_in = 5
    gaps_out = 20
    border_size = 3 # 2
    col.active_border = rgb(9E2A2B)
    col.inactive_border = rgba(59595900)

    layout = dwindle

    # Please see https://wiki.hyprland.org/Configuring/Tearing/ before you turn this on
    allow_tearing = false
}

decoration {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more

    rounding = 10
    
    blur {
        enabled = true
		new_optimizations = true
        size = 1
        passes = 6
    }

    drop_shadow = no
    shadow_range = 4
    shadow_render_power = 3
    col.shadow = rgba(1a1a1aee)
}

animations {
    enabled = yes

    # Some default animations, see https://wiki.hyprland.org/Configuring/Animations/ for more

    bezier = myBezier, 0.05, 0.9, 0.1, 1.05

    animation = windows, 1, 3, myBezier
    # animation = windows, 1, 7, myBezier
    animation = windowsOut, 1, 7, default, popin 80%
    animation = border, 1, 10, default
    animation = borderangle, 1, 8, default
    animation = fade, 1, 7, default
    animation = workspaces, 1, 2, default, slidevert
}

dwindle {
    # See https://wiki.hyprland.org/Configuring/Dwindle-Layout/ for more
    pseudotile = yes # master switch for pseudotiling. Enabling is bound to mainMod + P in the keybinds section below
    preserve_split = yes # you probably want this
}

master {
    # See https://wiki.hyprland.org/Configuring/Master-Layout/ for more
    # new_is_master = true
}

misc {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more
    force_default_wallpaper = 0 # Set to 0 or 1 to disable the anime mascot wallpapers
}



# bind = SUPER, grave, hyprexpo:expo, toggle # can be: toggle, off/disable or on/enable

plugin {
    hyprwinwrap {
        # class is an EXACT match and NOT a regex!
        class = kitty-bg
    }

    hyprexpo {
        columns = 3
        gap_size = 0
        bg_col = rgb(FFF3B0)
        workspace_method = first 1 # [center/first] [workspace] e.g. first 1 or center m+1

        enable_gesture = true # laptop touchpad, 4 fingers
        gesture_distance = 300 # how far is the "max"
        gesture_positive = true # positive = swipe down. Negative = swipe up.
    }
}

# Example windowrule v1
# windowrule = float, ^(kitty)$
# Example windowrule v2
# windowrulev2 = float,class:^(kitty)$,title:^(kitty)$
# See https://wiki.hyprland.org/Configuring/Window-Rules/ for more
# windowrulev2 = nomaximizerequest, class:.* # You'll probably like this.
windowrulev2 = pin, title:(.*)(Picture-in-Picture)(.*)
windowrulev2 = workspace 6, class:Spotify
windowrulev2 = workspace 7, class:Signal
windowrulev2 = workspace 2, class:Chromium
windowrulev2 = size 50% 50%, title:(.*)(Chromium)(.*)
# windowrulev2= opacity 0.85,class:^(Thorium)
# windowrulev2= opacity 0.85,title:(VCV)(.*)

# See https://wiki.hyprland.org/Configuring/Keywords/ for more
$mainMod = SUPER

# Example binds, see https://wiki.hyprland.org/Configuring/Binds/ for more
bind = $mainMod, Return, exec, $terminal --hold fastfetch
bind = $mainMod, Backspace, killactive,
bind = $mainMod, C, exec, hyprpicker -a
bind = $mainMod, F, fullscreen
bind = $mainMod, B, exec, $browser
bind = $mainMod, M, exit, 
bind = $mainMod, E, exec, $fileManager
bind = $mainMod, V, togglefloating,
bind = $mainMod, Space, exec, $menu
bind = $mainMod, P, pseudo, # dwindle
bind = $mainMod, H, togglesplit, # dwindle
bind = , Print, exec, grim -g "$(slurp -d)" - | swappy -f -
bind = $mainMod, D, exec, spotify-launcher
bind = $mainMod, Q, exec, xdg-open https://upload.wikimedia.org/wikipedia/commons/thumb/e/e6/KB_USA-standard.svg/400px-KB_USA-standard.svg.png # opens a querty keyboard
bind = $mainMod, T, exec, xdg-open https://translate.google.com/?sl=en&tl=fr # opens a traductor
bind = $mainMod, U, exec, $terminal $HOME/.config/scripts/rand_meme.sh
bind = $mainMod, N, exec, hyprshade toggle blue-light-filter

bind = $mainMod, Q, exec, hyprctl switchxkblayout at-translated-set-2-keyboard next

# Move focus with mainMod + arrow keys
bind = $mainMod, left, movefocus, l
bind = $mainMod, right, movefocus, r
bind = $mainMod, up, movefocus, u
bind = $mainMod, down, movefocus, d
# bind = $mainMod, Tab, hyprexpo:expo, toggle # can be: toggle, off/disable or on/enable

# Switch workspaces with mainMod + [0-9]
bind = $mainMod, code:10, workspace, 1
bind = $mainMod, code:11, workspace, 2
bind = $mainMod, code:12, workspace, 3
bind = $mainMod, code:13, workspace, 4
bind = $mainMod, code:14, workspace, 5
bind = $mainMod, code:15, workspace, 6
bind = $mainMod, code:16, workspace, 7
bind = $mainMod, code:17, workspace, 8
bind = $mainMod, code:18, workspace, 9
bind = $mainMod, code:19, workspace, 10

# Move active window to a workspace with mainMod + SHIFT + [0-9]
bind = $mainMod SHIFT, code:10, movetoworkspace, 1
bind = $mainMod SHIFT, code:11, movetoworkspace, 2
bind = $mainMod SHIFT, code:12, movetoworkspace, 3
bind = $mainMod SHIFT, code:13, movetoworkspace, 4
bind = $mainMod SHIFT, code:14, movetoworkspace, 5
bind = $mainMod SHIFT, code:15, movetoworkspace, 6
bind = $mainMod SHIFT, code:16, movetoworkspace, 7
bind = $mainMod SHIFT, code:17, movetoworkspace, 8
bind = $mainMod SHIFT, code:18, movetoworkspace, 9
bind = $mainMod SHIFT, code:19, movetoworkspace, 10

# Example special workspace (scratchpad)
bind = $mainMod, S, togglespecialworkspace, magic
bind = $mainMod SHIFT, S, movetoworkspace, special:magic

# Scroll through existing workspaces with mainMod + scroll
bind = $mainMod, mouse_down, workspace, e-1
bind = $mainMod, mouse_up, workspace, e+1

bind = $mainMod, page_up, workspace, e-1
bind = $mainMod, page_down, workspace, e+1

bind = $mainMod SHIFT, page_up, movetoworkspace, e-1
bind = $mainMod SHIFT, page_down, movetoworkspace, e+1

bind = $mainMod, K, workspace, -1
bind = $mainMod, J, workspace, +1

bind = $mainMod SHIFT, K, movetoworkspace, -1
bind = $mainMod SHIFT, J, movetoworkspace, +1

$player = spotify
bind =, XF86AudioPlay, exec, playerctl --player=$player play-pause
bind =, XF86AudioPrev, exec, playerctl --player=$player previous
bind =, XF86AudioNext, exec, playerctl --player=$player next
bind =, XF86AudioStop, exec, playerctl --player=$player stop

bind =, XF86Mail, exec, xdg-open https://outlook.live.com/ & xdg-open https://mail.google.com/ &

bind =, XF86HomePage, exec, hyprlock
bind =, XF86Sleep, exec, hyprlock

# Move/resize windows with mainMod + LMB/RMB and dragging
bindm = $mainMod, mouse:272, movewindow
bindm = $mainMod, mouse:273, resizewindow
bindm = $mainMod SHIFT, mouse:273, resizewindow 1

bind = $mainMod, KP_Left, resizeactive, -200 0
bind = $mainMod, KP_Right, resizeactive, 200 0
bind = $mainMod, KP_Up, resizeactive, 0 -200
bind = $mainMod, KP_Down, resizeactive, 0 200



debug:disable_logs = false

======Config-End========
```

</details>

### Description

When using keys to resize windows, Borders are late (see screenshot)
Binds :
```
bind = $mainMod, KP_Left, resizeactive, -200 0
bind = $mainMod, KP_Right, resizeactive, 200 0
bind = $mainMod, KP_Up, resizeactive, 0 -200
bind = $mainMod, KP_Down, resizeactive, 0 200
```

### How to reproduce

1. Add
```
bind = $mainMod, KP_Left, resizeactive, -200 0
bind = $mainMod, KP_Right, resizeactive, 200 0
bind = $mainMod, KP_Up, resizeactive, 0 -200
bind = $mainMod, KP_Down, resizeactive, 0 200
```
to your config
2. Resize !!

### Crash reports, logs, images, videos
(the blur is an edit)
![image](https://github.com/user-attachments/assets/42bf5a72-9463-45ca-a6ef-dc036273af7b)
Here, I see the border bugging at the right, after pressing SUPER + Keyboard 4 (left)

**Comment by mettz** [comment]
same issue here

**Comment by mettz** [comment]
was having the same problem while changing conf during update to hyprland 0.43. rebooting fixed the issue, @freerig maybe it can work for you too?

**Comment by freerig** [comment]
No, unfortunatly, reboot doesn't works for me. I still have to slooowlly move my hand to the mouse.

**Comment by vaxerski** [comment]
fixed in 0.43
